static/src/stylesheets/module/onward-garnett/_rich-links-default.head.scss

```diff
@@ -82,6 +82,13 @@
     margin: 5px $gs-gutter $gs-baseline 0;
     clear: both;
 
+    // Decouple rich link position from height of ad in RHS.
```

Comment:
Nitpick: We normally use `RHS` to refer to the RHS / article-aside slot... However, this issue relates to how to `inline-2` to `inline-N` refresh on the page. These slots get the following css applied:

```css
.ad-slot--offset-right {
    @include mq(desktop) {
        margin-right: -320px;
    }
```

This is how they move out of the article body at the desktop breakpoint 😸

Reply:
We should maybe try using the secondary column, but these slots all get inserted by our glorious overlord Spacefinder, and Spacefinder needs to position these slots based on the content of the main column... 🤔

Reply:
Makes sense, thanks for the clarity @katebee. I've updated the comment accordingly